ssl/internal.h

```diff
@@ -533,6 +533,9 @@ struct ssl_cipher_st {
   uint32_t algorithm_prf;
 };
 
+// print_all_supported_ciphers prints all supported ciphers to stdout.
+OPENSSL_EXPORT bool print_all_supported_ciphers(bool use_openssl_name);
```

Comment:
2 things:
1. Should this exported from an internal header?
2. NP: could this confuse ciphers (like AES) with what it is cipher suites?
```suggestion
bool ssl_print_all_supported_ciphersuites(bool use_openssl_name);
```
We seem to be split, all old functions use `ssl_` and newer internal functions use `tls_` or `tls1_` or `tls12_`.